client/settings-consumables.js

```diff
@@ -19,12 +19,22 @@ async function handleConsumableResetButton(type, subType) {
     }
 }
 
+function setConsumableData(element, res, type, subType) {
+    const data = res.find(e => e.type === type && e.subType === subType);
+    if (data) {
+        element.innerHTML = (data.remaining.value / 60).toFixed(1) + " hours left";
+    } else {
+        element.innerHTML = "N/A";
```

Comment:
I'd suggest hiding the elements by default and only making them visible if there is a value for them

Reply:
Okay, the list is getting generated dynamically now.

![image](https://user-images.githubusercontent.com/8963459/112023123-3c86eb00-8b33-11eb-8bee-d2d0a78fd637.png)

Reply:
Even better!